src/test/java/org/folio/rest/api/InventoryHierarchyViewTest.java

```diff
@@ -120,8 +121,10 @@ public void canRequestInventoryHierarchyHoldingsWithoutParameters() throws Inter
       instancesData.get(0),
       allOf(
         hasIdForHoldings(predefinedHoldings.getString("id")),
+        hasEffectiveLocationForHoldings("d:Main Library"),
         hasPermanentLocationForHoldings("d:Main Library"),
         hasPermanentLocationCodeForHoldings("TestBaseWI/M"),
+        hasEffectiveLocationCodeForHoldings("TestBaseWI/M"),
```

Comment:
To me, this only demonstrates that the effective location is the same as the permanent location. Personally, I might want to demonstrate that they could vary independently. With this check, the test could pass if the view fetched the permanent location for the effective location.

I think it is easier to follow if the order of the checks is the same, e.g. permanent and then effective location for each attribute. At the moment, they are in a different order for each attribute.

Reply:
I will refactor this and add an additional test looking specifically at what happens when the temporary location is set.